tests/test_multimodal_lm.py

```diff
@@ -46,44 +46,53 @@ def generate_text_simple(
             "llama2",
-            "Sustine et abstine condolences % contracted Expressku LookingOriginally laughable gathered redesign",
+            {
+                "llama": "Sustine et abstine staff пописа sur Datenrefref equations Soundantalfs",
+                "gpt2": "Sustine et abstine objections Sanskrit hormavin 25 noting carbs contamination chatting caramel",
+                "mini_gpt2": "Sustine et abstinelined regener cannabinoidibus fictional makeshift Literary Abdullah Tree dozens",
+            }
         ),
         (
             "gpt2",
-            "Sustine et abstinegreg LXamm Local addition Immun GlassrikeFal Resurrection",
+            {
+                "llama": "Sustine et abstinerefrefrefумrefnakeжёнfragmentrefwhy",
+                "gpt2": "Sustine et abstineohoorphLE updates� Oaks GD straight correlates Sir",
+                "mini_gpt2": "Sustine et abstine200 Qi heter Wallace revolution 1997 349ety PARK682",
+            }
         ),
     ],
 )
 def test_multimodal_llm(backend_target):
     torch.manual_seed(999)
     backend, target = backend_target
-    tokenizer = GPT2Tokenizer()
-    model = MultiModalLM(
-        settings=MultiModalLMSettings(
-            vision_input_shape=(3, 3, 2, 1),
-            backend=backend,
-            n_heads=1,
-            n_layers=1,
-            emb_dim=32,
-            hidden_dim=32,
-            context_length=32,
+    for tokenizer in [LlamaTokenizer(), GPT2Tokenizer(), MiniTokenizer(GPT2Tokenizer())]:
+        model = MultiModalLM(
+            settings=MultiModalLMSettings(
+                vision_input_shape=(3, 3, 2, 1),
+                backend=backend,
+                n_heads=1,
+                n_layers=1,
+                emb_dim=32,
+                hidden_dim=32,
+                context_length=32,
+            )
         )
-    )
-    vision_input = NamedTensor(
-        torch.randn(1, 3, 3, 2, 1),
-        names=("batch", "lat", "lon", "timestep", "features"),
-        feature_names=("u",),
-    )
-    encoded = tokenizer.encode("Sustine et abstine")
-    encoded_tensor = torch.tensor(encoded).unsqueeze(0)
+        vision_input = NamedTensor(
+            torch.randn(1, 3, 3, 2, 1),
+            names=("batch", "lat", "lon", "timestep", "features"),
+            feature_names=("u",),
+        )
+        encoded = tokenizer.encode("Sustine et abstine")
+        encoded_tensor = torch.tensor(encoded).unsqueeze(0)
 
-    out = generate_text_simple(
-        model=model,
-        idx=encoded_tensor,
-        max_new_tokens=10,
-        context_size=model.context_length,
-        vision_input=vision_input,
-    )
-    decoded_text = tokenizer.decode(out.squeeze(0).tolist())
+        out = generate_text_simple(
+            model=model,
+            idx=encoded_tensor,
+            max_new_tokens=10,
+            context_size=model.context_length,
+            vision_input=vision_input,
+        )
+        out = torch.where(out > tokenizer.vocab_size-1, 999, out)
```

Comment:
```suggestion
        out = torch.where(out > tokenizer.vocab_size - 1, 999, out)
```
Did you run the linter ?

Reply:
Of course not ! ^^